views/templates/partials/nav.ejs

```diff
@@ -0,0 +1,15 @@
+<nav class="text-gray-700 bg-gray-400 body-font m">
+    <ul class="container mx-auto flex flex-row-reverse p-10 md:flex-row-reverse ">
+        <%if (isLoggedIn) { 
+            %>
+            <li class="mx-2"><a href="#">Log Out</a></li>
+            <li class="mx-2"><a href="#">Profile</a></li>
+            <li class="mx-2"><a href="#">Artist</a></li>
+            <li class="mx-2"><a href="/concert-search">Concert</a></li>
+       
```

Comment:
could we add a tab to also get back to the home / landing page?

Reply:
yeah, I can do it. I think that tap should be accessible to everyone like authenticated or not can see that tap

Reply:
yeah that makes sense